lib/build.api.js

```diff
@@ -70,7 +69,7 @@ class Build {
       let project = this.aquifer.project;
 
       // Set some reasonable defaults.
-      this.options.makeFile = this.getAbsolutePath(this.options.makeFile) || project.absolutePaths.makeFile;
+      this.options.makeFile = project.absolutePaths.make;
```

Comment:
I just remembered ...
```
this.getAbsolutePath(this.options.makeFile)
```
This fails without providing a real path (`false` isn't a real path)